services/packagist/packagist-helpers.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const {
+  compare: phpVersionCompare,
+  latest: phpLatestVersion,
+  isStable: phpStableVersion,
+} = require('../../lib/php-version')
+
+const getLatestVersion = function(versionsData, stable = true) {
+  let versions = Object.keys(versionsData)
+
+  // Map aliases (eg, dev-master).
+  const aliasesMap = {}
+  versions.forEach(version => {
+    const versionData = versionsData[version]
+    if (
+      versionData.extra &&
+      versionData.extra['branch-alias'] &&
+      versionData.extra['branch-alias'][version]
+    ) {
```

Comment:
This is fine, though you could write the above even more functionally like this:

```js
const aliasesMap = {}
Object.entries(versionsData)
  .filter(
    (version, versionData) =>
      versionData.extra &&
      versionData.extra['branch-alias'] &&
      versionData.extra['branch-alias'][version]
  )
  .forEach((version, versionData) => {
    ...
```